tests/Meta_test.ml

```diff
@@ -241,13 +244,25 @@ let remove_optional_lines =
 
 let mask_and_sort = mask_alcotest_output @ [ sort_lines; remove_optional_lines ]
 
+(* FIXME: Running parallel jobs on Windows is unstable, producing
+   Sys_error("Invalid argument")` during some attempts to flush channels.
+   Perhaps related to https://github.com/ocaml/ocaml/issues/13586 *)
+let skipped =
+  if Sys.win32 then
+    Some "Running tests in parallel is unstable on Windows"
+  else
+    None
```

Comment:
This is strange. Testo doesn't use threads, all parallelism is done with multiprocessing.

Reply:
Agreed that it is strange! I'm hoping we can investigate at some point, tho don't have the bandwidth for it as part of this work unfortunately.